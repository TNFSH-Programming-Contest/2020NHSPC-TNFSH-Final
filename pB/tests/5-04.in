4
436.316 192.46
159.144 192.46
159.144 -237.352
436.316 -237.352
548.425 -626.95
