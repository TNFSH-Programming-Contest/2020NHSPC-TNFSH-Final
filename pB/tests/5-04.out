2069107.634
